src/assets/css/global/_main.scss

```diff
@@ -395,6 +395,10 @@ hr {
       }
     }
 
+    input {
+      text-align: right;
```

Comment:
Why do we need it?

Reply:
so the number showed to the right like in the pic 

![156377386-ab8940d3-bffe-408f-aa2c-3e5c59ecb087](https://user-images.githubusercontent.com/85109141/157401600-2233904d-0846-436c-9764-2035f44fd1b3.png)